cuda/cuda11.1/build_image.sh

```diff
@@ -0,0 +1,2 @@
+DOCKER_BUILDKIT=1 \
+  docker build -t ghcr.io/neuralmagic/base-cuda11.1 .
```

Comment:
Let's also explicitly pass in CUDA_VERSION build-arg, that way if we need another version of CUDA it would just be a one word change?

Reply:
sure but since we dont seem to change the cuda version we use frequently, maybe let's just start with the one we currently need?